cold-start/runtime-environment/SAND/README.md

```diff
@@ -0,0 +1,39 @@
+# **[SAND: Towards High-Performance Serverless Computing](https://www.usenix.org/system/files/conference/atc18/atc18-akkus.pdf)**
+
```

Comment:
Coloca uma linha "Abstract" no início, pensei que já era parte do seu resumo

Reply:
Done.